actiontext/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+*   Dispatch direct-upload events on attachment uploads
+
+    When using Action Text's rich textarea,  it's possible to attach files to the
+    editor. Previously, that action didn't dispatch any events, which made it hard
+    to react to the file uploads. For instance, if an upload failed, there was no
+    way to notify the user about it, or remove the attachment from the editor.
+
+    This commits adds new events - `direct-upload:start`, `direct-upload:progress`,
```

Comment:
Should we document those events like we do with the Active Storage ones?

Reply:
Good question! I was wondering if I should put the docs in the Active Storage or in Action Text guide. I guess Action Text makes more sense?

Reply:
I think so.

Reply:
Added!